cognite/neat/_graph/extractors/_classic_cdf/_base.py

```diff
@@ -87,7 +87,6 @@ def __init__(
         self,
         items: Iterable[T_CogniteResource],
         namespace: Namespace | None = None,
-        to_type: Callable[[T_CogniteResource], str | None] | None = None,
```

Comment:
Removing this unused feature after discussion with @nikokaoja . Fixing the parent issue becomes more complicated with it, and decided not to continue with it.